src/orchestrator/types.ts

```diff
@@ -282,6 +285,50 @@ interface TokenConfig {
   balanceSlot: (address: Address) => Hex
 }
 
+export enum MainnetNetwork {
+  ETHEREUM = 1,
+  OPTIMISM = 10,
+  POLYGON = 137,
+  BASE = 8453,
+  ARBITRUM = 42161,
+}
+
+export enum TestnetNetwork {
+  POLYGON_AMOY = 80002,
+  BASE_SEPOLIA = 84532,
+  ARBITRUM_SEPOLIA = 421614,
+  ETHEREUM_SEPOLIA = 11155111,
+  OPTIMISM_SEPOLIA = 11155420,
+}
+
+export enum TokenSymbol {
+  ETH = 'ETH',
```

Comment:
Could we use the union types that we have:

```
SupportedTestnet
SupportedMainnet
SupportedTokenSymbol
```

Will also address @tadovas comment